.gitignore

```diff
@@ -24,6 +24,12 @@ wheels/
 .installed.cfg
 *.egg
 MANIFEST
+.DS_Store
+examples/.DS_Store
+
+#Other gui
+examples/kivy/
```

Comment:
If this is something unique to your local experimentation, let's not add it to the repo.

Reply:
I'll remove it.